labrad/decorators.py

```diff
@@ -28,6 +28,7 @@
 import types
 
 import twisted.internet.defer as defer
+from past.builtins import basestring  # for python 2/3 compatibility
```

Comment:
I tried importing this in 2.7 -- it looks like it's third party. Is it a part of the `future` library?